src/test/java/io/pravega/connectors/flink/utils/SetupUtils.java

```diff
@@ -304,4 +328,22 @@ public ClientConfig getClientConfig() {
                     .build();
         }
     }
+
+    /*
+     * A utility class to set flags for tests against secure Pravega.
+     */
+    public static class SetupUtilsUpdate {
```

Comment:
Why not simply enhance `SetupUtils` to provide this functionality directly?  Like, provide public setters that must be called before `.start()`.

Reply:
Done.